packages/create-razzle-app/lib/examples.js

```diff
@@ -0,0 +1 @@
+module.exports = ["basic","basic-spa","with-afterjs","with-custom-babel-config","with-custom-devserver-options","with-custom-environment-variables","with-custom-webpack-config","with-elm","with-firebase-functions","with-heroku","with-hyperapp","with-inferno","with-jest-snapshots","with-jsxstyle","with-koa","with-less","with-loadable-components","with-material-ui","with-mdx","with-now","with-now-v2","with-polka","with-preact","with-rax","with-react-loadable","with-react-native-web","with-react-router-3","with-reason-react","with-redux","with-redux-observable","with-scss","with-styled-components","with-typescript","with-vendor-bundle","with-vue","with-vue-router"];
```

Comment:
it's not good to hard code example names
why this can't be dynamic?

Reply:
we can use Axios with GitHub API to check if one example is available or not

```js
const repoOwner = "jaredpalmer"
const repoName = "razzle"
// don't use "/" at start :)
const examplesDirectoryPath = "examples"

const isFolder = ({ type }) => type === "dir"
const { data } = await axios.get(`https://api.github.com/repos/${repoOwner}/${repoName}/contents/${examplesDirectoryPath}`)
const validExampleNames = data.filter(isFolder).map(({ name }) => name)
```

and this is how `validExmapleNames` will look like at the end
```js
["basic-spa", "basic", "with-afterjs", "with-custom-babel-config", "with-custom-devserver-options", "with-custom-environment-variables", "with-custom-webpack-config", "with-elm", "with-firebase-functions", "with-heroku", "with-hyperapp", "with-inferno", "with-jest-snapshots", "with-jsxstyle", "with-koa", "with-loadable-components", "with-material-ui", "with-mdx", "with-now-v2", "with-now", "with-polka", "with-preact", "with-rax", "with-react-loadable", "with-react-native-web", "with-react-router", "with-reason-react", "with-redux", "with-scss", "with-styled-components", "with-typescript", "with-vendor-bundle", "with-vue-router", "with-vue"]
 ```

more info:
https://developer.github.com/v3/repos/contents/
https://api.github.com/repos/jaredpalmer/razzle/contents/examples

Reply:
Because how do we distinguish official examples from npm examples. npm: prefix? But then we differ from create-react-app. Or move examples into create-razzle-app, but then that becomes large to load with npx. So prefix or hardcode.

Reply:
I think it's better to add a prefix for npm
but we can call the GitHub API at first and then we look for the example if we didn't find it then we try NPM ...

Reply:
Good idea :)